### PR TITLE
Item 8418: Migrate FDAMyStudies repo to new github home

### PR DESCRIPTION
**Pull request by labkey-tchad**
#### Rationale
Gradle requires project names to match the directory name so we need to update some gradle paths. For consistency, we are renaming some other aspects of the "mobileAppStudy" module to match the new "Response" name.

#### Related Pull Requests
* LabKey/platform#2006
* LabKey/testAutomation#641
* LabKey/distributions#152 (merged)

#### Changes
* Rename module: "MobileAppStudy" -> "Response"
* Rename gradle project: ":server:modules:mobileAppStudy" -> ":server:modules:Response"
* Rename folder type:  "Mobile App Study" -> "MyStudies Response"
* Rename webpart: "Mobile App Study Setup" -> "MyStudies Study Setup"
* Refactor `ReadResponseTest` to do more setup via API

### CODE REVIEW

File: resources/folderTypes/response.foldertype.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<folderType xmlns="http://labkey.org/data/xml/folderType" xmlns:mp="http://labkey.org/moduleProperties/xml/">
-    <name>Mobile App Study</name>
-    <description>The default folder type to support Mobile App Study Data Management and Analysis</description>
+<folderType xmlns="http://labkey.org/data/xml/folderType">
+    <name>MyStudies Responses</name>
```
Comment (labkey-adam) [inline resources/folderTypes/response.foldertype.xml]:
```suggestion
    <name>MyStudies Response</name>
```

File: src/org/labkey/mobileappstudy/MobileAppStudyModule.java [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
 public class MobileAppStudyModule extends DefaultModule
```
Comment (labkey-adam) [inline src/org/labkey/mobileappstudy/MobileAppStudyModule.java]: Might as well change the class name. This means updating module.properties and the core.Modules update query as well.

**Review by labkey-adam — APPROVED** [review]
Couple suggestions. Looks like several tests are failing, but I'm sure you'll fix those.

File: README.md [hunk 3]
```diff
@@ -9,28 +9,22 @@ This custom module provides the Response Server functionality and services, incl
...
-1. Navigate to the root of your SVN working copy (the level right above the "server" directory).
-1. In your settings.gradle file, find the commented out line with this text:
-
-    ```
-    //include ":server:modules:workflow"
-    ```
-
-1. Underneath this line, add these two lines:
+1. Checkout the LabKey Server 21.3.x public GitHub repositories: [Set Up a Development Machine](https://www.labkey.org/Documentation/wiki-page.view?name=devMachine)
+1. Navigate to the root of your LabKey enlistment.
```
Comment (labkey-adam) [inline README.md:18]: Add a line about cloning the FDA-MyStudies/Response GitHub repo into the enlistment?

File: src/org/labkey/mobileappstudy/MobileAppStudyModule.java (hunk 2 above)
Comment (labkey-tchad) [reply src/org/labkey/mobileappstudy/MobileAppStudyModule.java]: Updated all package and non-schema class names to `Response*`. I did not change any JavaScript `LABKEY.mobilaAppStudy` names.

**Pull request merged**